plugin/manager.go

```diff
@@ -289,13 +290,46 @@ func attachToLog(id string) func(libcontainerd.IOPipe) error {
 }
 
 func validatePrivileges(requiredPrivileges, privileges types.PluginPrivileges) error {
-	// todo: make a better function that doesn't check order
-	if !reflect.DeepEqual(privileges, requiredPrivileges) {
+	if !isEqual(requiredPrivileges, privileges, isEqualPrivilege) {
 		return errors.New("incorrect privileges")
 	}
+
 	return nil
 }
 
+func isEqual(arrOne, arrOther types.PluginPrivileges, compare func(x, y types.PluginPrivilege) bool) bool {
```

Comment:
In addition, `Values` is also a list, so we need to sort both `PluginPrivileges` lists as so as `Values` lists in each `PluginPrivilege`. So I think using the function `isEqual` may be more simple and clear.